### ISSUE TITLE
Issues and PR templates

### ISSUE THREAD

**Issue by benjohn88** [Xwilarg/TwitterSharp#60]
I feel having an Issue template would be awesome and really helpful to report bug, suggest feature etc. This will also make sure that there is always sufficient details added when an issue is reported or  when a PR is raised. 

What are your thoughts? Thanks

**Comment by marcogruhl** [comment]
Good idea, could you provide one(multiple)? At least @Xwilarg had to implement it. I also would keep it very simple.

**Comment by Xwilarg** [comment]
GitHub gives some templates we could use but well, since @marcogruhl only have the free Twitter plan and I don't have access to it at all, I'm not sure if it would be really necessary
